tableau_utilities/hyper/hyper.py

```diff
@@ -0,0 +1,112 @@
+import os
+import shutil
+from zipfile import ZipFile
+from tableauhyperapi import HyperProcess, Connection, Telemetry, CreateMode, TableDefinition, TableName, SqlType
+
+from tableau_utilities.tableau_file.tableau_file import TableauFileError
+
+
+def create_empty_hyper_extract(datasource: Datasource):
+    """ Creates an empty extract (.hyper file) for the Tableau file.
+        If the extract exists, it will be overwritten.
+        Args:
+            Datasource: The tableau_utilities datasource class
+    """
+    # Get relevant paths, and create a temp folder and move the Tableau file into it
+    temp_folder = os.path.join(Datasource.file_directory, f'__TEMP_{Datasource.file_name}')
```

Comment:
```suggestion
    temp_folder = os.path.join(datasource.file_directory, f'__TEMP_{Datasource.file_name}')
```
All of these references still need to be updated